tests/functional/spec/preview.spec.js

```diff
@@ -1,13 +1,27 @@
 import IntroductionPage from "../generated_pages/introduction_preview_hub/introduction.page";
+import IntroductionPageLinear from "../generated_pages/introduction/introduction.page";
 
 describe("Introduction preview questions", () => {
   const introductionSchema = "test_introduction_preview_hub.json";
+  const introductionSchemaLinear = "test_introduction.json";
   const showButton = 'button[data-ga-category="Preview Survey"]';
   const previewSummaryContent = "#summary-accordion-1-content";
-  beforeEach(() => {
-    browser.openQuestionnaire(introductionSchema);
-  });
+  const previewSectionTitle = ".ons-summary__group-title";
+  const previewQuestion = ".ons-summary__item";
 
+  it("Given I start a survey, When I view the preview page, Then all preview elements should be visible", () => {
+    browser.openQuestionnaire(introductionSchemaLinear);
+    $(IntroductionPageLinear.previewQuestions()).click();
+    expect(browser.getUrl()).to.contain("questionnaire/preview");
+    expect($(previewSectionTitle).getText()).to.equal("Main section");
+    expect($(previewQuestion).$("h3").getText()).to.equal("Are you able to report for the calendar month 1 January 2017 to 1 February 2017?");
+    expect($(previewQuestion).$(".ons-question__description").getText()).to.equal("Your return should relate to the calendar year 2021.");
+    expect($(previewQuestion).$(".ons-question__instruction").getText()).to.equal("Select your answer");
+    expect($(previewQuestion).$(".ons-panel__body").getText()).to.equal("You can only pick one answer");
+    expect($(previewQuestion).$$("p")[3].getText()).to.equal("Choose Yes or No answer");
+    expect($(previewQuestion).$$("p")[4].getText()).to.equal("You can answer with the following options:");
+    expect($(previewQuestion).$("ul").getText()).to.equal("Yes\nNo");
```

Comment:
Being pragmatic but these should use data attrs not raw classes and tags. It makes it very fragile when DS changes or classes update etc.

Reply:
Ok will add comment.